App/CourseApp/SharedUI/TextType.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  TextType.swift
+//  CourseApp
+//
+//  Created by Marcel Mravec on 17.05.2024.
+//
+
+import SwiftUI
+import UIKit
+
+enum FontSize: CGFloat {
+    case size36 = 36
+    case size28 = 28
+    case size20 = 20
+    case size12 = 12
+}
+
+enum FontType: String {
+    case regular = "Poppins-Regular"
+    case bold = "Poppins-Bold"
+    case mediumItalic = "Poppins-MediumItalic"
+}
+
+struct TextTypeModifier: ViewModifier {
```

Comment:
I would split the file into multiple. For every type of view modifier I would go with individual file.